### ISSUE TITLE
How can I avoid usage of fonts which have an MIT license?

### ISSUE THREAD

**Issue by jaygazula27** [OfficeDev/office-ui-fabric-react#618]
I'm really liking this collection of components, but I do have a quick question about the fonts license.

In the README, it says that the files in this repo are subject to the MIT license. But it also goes on to say that the fonts referenced in these files have a different license.
How can I make use of these components but at the same time not accidentally make use of the fonts with the other license?

Basically, I only want stick to the stuff that has the MIT license. I figured I get clarification before going all in on this library.

**Comment by mikewheaton** [comment]
As you've seen, usage of the Segoe UI web font (as well as icons, brand logos, etc.) is subject to the [Fabric assets license](https://static2.sharepointonline.com/files/fabric/assets/license.txt).

A new version of this license will be released shortly but the basic premise is the same: you may only use these fonts and icons for projects that "extend the Office User Experience including: Office Add-Ins (previously known as Apps for Office), SharePoint Apps or Add-Ins and Office 365 Apps (those launched from within the Office launcher)". We generally find this to be inclusive enough for developers who are interested in Fabric, as using components that follow the Office Design Language is of less value if you're developing outside of the Microsoft ecosystem.

You could write CSS overrides to use different fonts than are specified in [Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core). This would be relatively straightforward for replacing Segoe UI (you could substitute for Arial or Helvetica, perhaps) but more tricky for the icon font, as you'd need to map every icon class (e.g. `ms-Icon--Cancel`) to an equivalent symbol in another font.

It's recently come to our attention through [another issue](https://github.com/OfficeDev/office-ui-fabric-core/issues/852) that there may be [open source alternatives to Microsoft fonts](https://github.com/Microsoft/fonts) but our team has not yet had time to investigate and discuss this.

**Comment by jaygazula27** [comment]
@mikewheaton Thank you so much for your response!
I think it would be best if I just stay on the safe side for now. I'm afraid that it might be too easy for me to miss an override which results in me breaking the license.

I can only hope that the open source alternatives are used. But nonetheless, I've loved these components thus far!
